.github/workflows/ci.yml

```diff
@@ -7,10 +7,10 @@ jobs:
       - uses: actions/checkout@v2
       - uses: olafurpg/setup-scala@v10
       - run: ./scalafmt --test
-  build:
+  migration-rules:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: olafurpg/setup-scala@v10
       - run: sbt test
-      - run: cd scalafix && sbt test
+      - run: cd migration-rules && sbt test
```

Comment:
I think there is something off in the layout, https://github.com/scalacenter/scalafix.g8/tree/9073804c25967bb94dc2b13c75d3d42da724a753/migration-rules should only contain 2 directories, and this should iterate over them